applications/nrf5340_audio/boards/nrf5340_audio_dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <dt-bindings/led/led.h>
+
+/ {
+	audioleds: audioleds {
+		compatible = "gpio-leds";
+		status = "okay";
+
+		led_device_type: led_device_type {
+			gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>,
```

Comment:
I understood that you @nordicjm and Graham discussed this. The IMHO last piece of the puzzle would be to get this inherited from the original boardfile. If not possible, we will leave as-is.

Reply:
upstream it?

Reply:
Possible, but even if we did, it seems that this line `gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>,` will exist twice.